gstchecksumsink.c

```diff
@@ -157,6 +166,8 @@ gst_cksum_image_sink_init (GstCksumImageSink * checksumsink)
   gst_base_sink_set_sync (GST_BASE_SINK (checksumsink), FALSE);
   checksumsink->hash = G_CHECKSUM_MD5;
   checksumsink->frame_checksum = TRUE;
+  checksumsink->raw_file_name = NULL;
```

Comment:
Not need to set it to NULL, since when a gobject is created it's content is zeroed.

Reply:
Ok, I will remove and update PR after your responses for the other two comments

Reply:
dah! true... you didn't add a new variable :)

Reply:
Wait, `raw_file_name` is a `gchar *`... IIUC, gchar is just a typedef to char, so nothing is created or initialized.

Reply:
...and since we inspect it in `open_raw_file`, we need to set to NULL initially.

Reply:
raw pointers can have arbitrary value... so needs to be set to NULL.

Reply:
dah! of course... `raw_file_name` is part of `GstCksumImageSink` gobject... sorry for the noise.  Guess I should've had my coffee first ;)